sound/soc/sof/intel/hda-ctrl.c

```diff
@@ -284,10 +284,6 @@ void hda_dsp_ctrl_stop_chip(struct snd_sof_dev *sdev)
 	snd_sof_dsp_update_bits(sdev, HDA_DSP_HDA_BAR, SOF_HDA_INTCTL,
 				SOF_HDA_INT_ALL_STREAM,	0);
 
-	/* disable controller CIE and GIE */
-	snd_sof_dsp_update_bits(sdev, HDA_DSP_HDA_BAR, SOF_HDA_INTCTL,
-				SOF_HDA_INT_CTRL_EN | SOF_HDA_INT_GLOBAL_EN,
-				0);
```

Comment:
Question is does snd_sof_dsp_write need the interrupts?

Reply:
I don't know if this is useful, what I was thinking is that disabling the interrupts and then stopping the commands makes no sense in general. If there was an in-flight command it would be blocked, be it with the CORB/RIRB or PIO mode.

For now this doesn't have any effect so we can drop it. Still it'd be good to double-check this.

Reply:
The PIO mode is polling on the IC registers, it is not using interrupts, CORB/RIRB uses interrupts.